batch/batch/driver/instance_pool.py

```diff
@@ -657,49 +666,52 @@ async def handle_call_delete_event(self, instance, timestamp):
         await instance.mark_deleted('deleted', timestamp)
 
     async def handle_event(self, event):
-        payload = event.get('jsonPayload')
+        payload = event.get('protoPayload')
         if payload is None:
             log.warning('event has no payload')
             return
 
         timestamp = dateutil.parser.isoparse(event['timestamp']).timestamp() * 1000
-        version = payload['version']
-        if version != '1.2':
-            log.warning('unknown event verison {version}')
-            return
 
         resource_type = event['resource']['type']
         if resource_type != 'gce_instance':
             log.warning(f'unknown event resource type {resource_type}')
             return
 
-        event_type = payload['event_type']
-        event_subtype = payload['event_subtype']
-        resource = payload['resource']
-        name = resource['name']
+        if 'last' in event['operation']:
+            event_type = 'GCE_OPERATION_DONE'
+        elif 'first' in event['operation']:
+            event_type = 'GCE_API_CALL'
+        else:
+            log.warning(f'unknown event_type {event["operation"]}')
+            event_type = None
```

Comment:
We have the opportunity now to use more sensible names than these two. Maybe we can use `STARTED` and `COMPLETE` instead of GCE_API_CALL and GCE_OPERATION_DONE?